American Whitewater/AppDelegate.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  AppDelegate.swift
+//  American Whitewater
+//
+//  Updated by David Nelson on 7/4/19.
+//  Copyright © 2019 American Whitewater. All rights reserved.
+//
+
+import UIKit
+import CoreData
+import Firebase
+import OneSignal
+import IQKeyboardManagerSwift
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    var window: UIWindow?
+    
+    // setup CoreData functions and properties
+    lazy var persistentContainer: NSPersistentContainer = {
+        let container = NSPersistentContainer(name: "AmericanWhitewaterV2")
+        container.loadPersistentStores(completionHandler: { (storeDescription, error) in
+            print(storeDescription)
+            if let error = error {
+                fatalError("Unresolved Error: \(error), \(error.localizedDescription)")
+            }
+        })
+        
+        return container
+    }()
+    
+    func saveAWContext() {
+        let context = persistentContainer.viewContext
+        if context.hasChanges {
+            do {
+                try context.save()
+            } catch {
+                let error = error as NSError
+                fatalError("Unresolved error: \(error), \(error.userInfo)")
+            }
+        }
+    }
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        
+        // setup OneSignal for push notifications
+        // NOTE: only works on actual devices!
+        let onesignalInitSettings = [kOSSettingsKeyAutoPrompt: false]
+
+        // Replace 'YOUR_APP_ID' with your OneSignal App ID.
+        OneSignal.initWithLaunchOptions(launchOptions, appId: "fd159b28-68c6-465e-9301-1bf712f6c435",
```

Comment:
There are no security concerns with having this id committed right?

Reply:
I removed this. The previous key was a test key so not a security risk.